cardano-testnet/src/Testnet/Components/Configuration.hs

```diff
@@ -137,8 +137,8 @@ createSPOGenesisAndFiles (NumPools numPoolNodes) era shelleyGenesis alonzoGenesi
     , "--spec-conway",  inputGenesisConwayFp
     , "--testnet-magic", show testnetMagic
     , "--pools", show numPoolNodes
-    , "--total-supply",     show @Int 2_000_000_000_000
-    , "--delegated-supply", show @Int 1_000_000_000_000
+    , "--total-supply",     show @Int 2_000_000_000_000 -- 2 trillions
```

Comment:
So we are now overwriting the `inputGenesisShelleyFp`. We don't want to do this.

Reply:
It would be better to either:
- Disallow values that result in a negative treasury
- Add a check that will fail if the treasury in the node state is negative

Reply:
@Jimbo4350> Please note that this was the case before this PR, I'm not changing this behavior here :slightly_smiling_face:

> Disallow values that result in a negative treasury

> Add a check that will fail if the treasury in the node state is negative

That was actually my first instinct, but alas, looking at the ledger's spec; it seems quite involved to catch cases that will cause treasury to be negative (at least with my level of knowledge) and also it seemed wrong to have this code here. I would favor reusing the ledger's core that does this (if any).

> We don't to hard code values unless its specifically contained in a default Haskell value

Agreed, but I would rather do that in a follow-up PR. This PR is an improvement over the current state, so it makes sense as a first step.

Reply:
> Please note that this was the case before this PR, I'm not changing this behavior here 

Yep definitely wasn't blaming you. It was likely me that put those values there :clown_face:

> Agreed, but I would rather do that in a follow-up PR.

:+1: